templates/contribute/style.md

```diff
@@ -503,8 +503,16 @@ example {x y : ℝ} (hxy : x ≤ y) (h : ∀ ε > 0, y - ε ≤ x) : x = y :=
     linarith  
 ```
 
-Both when using the rewrite tactic (`rw`) and the `do` notation, there should
-be a space after the left arrow (`←`).
+When using the rewrite tactic `rw`, there should be a space after the left arrow (`←`):
+
+```lean
```

Comment:
I was thinking of a much shorter inline example like `rw [← add_comm a b]`

Reply:
And for the do notation you could use `do (← f) + (← g)`

Reply:
Done